internal/kafka/internal/services/kafkatlscertmgmt/secure_storage.go

```diff
@@ -121,7 +121,7 @@ func (storage *secureStorage) Exists(ctx context.Context, key string) bool {
 
 func (storage *secureStorage) List(ctx context.Context, prefix string, recursive bool) ([]string, error) {
 	filterKey := "name"
-	filterValue := fmt.Sprintf("%s/", storage.secretPrefix)
+	filterValue := storage.constructSecretName(prefix)
```

Comment:
Is `prefix` an appropriate name in this case?
Other methods that call `constructSecretName` use `key`, as the prefix is what is prefixed before within the `constructSecretName` method by using `storage.secretPrefix`